src/mixins/I13nMixin.js

```diff
@@ -148,8 +157,87 @@ var I13nMixin = {
         if (IS_DEBUG_MODE) {
             this._debugDashboard.destroy();
         }
+
+        this._removeSubComponentsListenersAndDebugDashboards();
+    },
+  
+    /**
+     * scan links
+     * @method _scanLinks
+     * @private
+     */
+    _scanLinks: function () {
+        var self = this;
+        var DOMNode = self.getDOMNode();
+        var foundElements = [];
+        var reactI13n = self._getReactI13n();
+        var scanTags = (self.props.scanLinks && self.props.scanLinks.tags) || DEFAULT_SCAN_TAGS;
+        if (!DOMNode) {
+            return;
+        }
+        self._subI13nComponents = [];
+
+        // find all links
+        scanTags.forEach(function scanElements(tagName) {
+            var collections = DOMNode.getElementsByTagName(tagName);
+            if (collections) {
+                foundElements = foundElements.concat(Array.prototype.slice.call(collections));
+            }
+        })
+
+        // for each link
+        // 1. create a i13n node
+        // 2. bind the click event
+        // 3. fire created event
+        foundElements.forEach(function registerFoundElement(element) {
+            var I13nNode = reactI13n.getI13nNodeClass();
+            var i13nNode = new I13nNode(self._i13nNode, {}, true, reactI13n.isViewportEnabled());
+            i13nNode.setDOMNode(element);
+            self._subI13nComponents.push({
+                componentClickHandler: EventListener.listen(element, 'click', clickHandler.bind(objectAssign({}, self, {_i13nNode: i13nNode}))),
+                i13nNode: i13nNode,
+                debugDashboard: IS_DEBUG_MODE ? new DebugDashboard(i13nNode) : null
+            });
+            self._getReactI13n().execute('created', {i13nNode: i13nNode});
+        });
+    },
+
+    /**
+     * _subComponentsViewportDetection, will be executed by viewport mixin
+     * @method _subComponentsViewportDetection
+     * @private
+     */
+    _subComponentsViewportDetection: function () {
+        var self = this;
+        if (self._subI13nComponents && 0 < self._subI13nComponents.length) {
+            self._subI13nComponents.forEach(function detectSubComponentViewport(subI13nComponent) {
+                self._detectElement(subI13nComponent.i13nNode, function enterViewportCallback() {
+                    subI13nComponent.i13nNode.setIsInViewport(true);
+                    self._getReactI13n().execute('enterViewport', {
+                        i13nNode: subI13nComponent.i13nNode
+                    });
+                });
```

Comment:
yes for i13n case, we only handle the `enterViewport` for firing linkviews, and we don't integrate with exitViewport for now.